libs/redux/userTerrains/api.ts

```diff
@@ -0,0 +1,14 @@
+import clientApi from '../clientApi';
+
+const UserTerriansAPI = clientApi.injectEndpoints({
+  endpoints: (build) => ({
+    getuserTerrians: build.query({
```

Comment:
Problem 1:
Spelling mistake, `getuserTerrians` should be `getuserTerrains`

Problem 2:
CamelCase naming convention `getuserTerrains` should be `getUserTerrains`